master/internal/checkpoints/postgres_checkpoints_intg_test.go

```diff
@@ -229,65 +448,38 @@ func TestDeleteCheckpoints(t *testing.T) {
 		Name:       "checkpoint 2",
 		Comment:    "empty",
 	}
-	err = db.QueryProto(
+	err = db.SingleDB().QueryProto(
 		"insert_model_version", &mv, pmdl.Id, retCkpt2.Uuid, addmv.Name, addmv.Comment,
 		emptyMetadata, strings.Join(addmv.Labels, ","), addmv.Notes, user.ID,
 	)
 	require.NoError(t, err)
 
-	// Test CheckpointsByUUIDs
-	reqCheckpointUUIDs := []uuid.UUID{checkpoint1.UUID, checkpoint2.UUID, checkpoint3.UUID}
-	checkpointsByUUIDs, err := db.CheckpointByUUIDs(reqCheckpointUUIDs)
-	require.NoError(t, err)
-	dbCheckpointsUUIDs := []uuid.UUID{
-		*checkpointsByUUIDs[0].UUID, *checkpointsByUUIDs[1].UUID, *checkpointsByUUIDs[2].UUID,
-	}
-	sortUUIDSlice(reqCheckpointUUIDs)
-	sortUUIDSlice(dbCheckpointsUUIDs)
-	require.Equal(t, reqCheckpointUUIDs, dbCheckpointsUUIDs)
-
-	// Test GetModelIDsAssociatedWithCheckpoint
-	expmodelIDsCheckpoint := []int32{pmdl.Id}
-	modelIDsCheckpoint, err := GetModelIDsAssociatedWithCheckpoint(context.TODO(), checkpoint1.UUID)
-	require.NoError(t, err)
-	require.Equal(t, expmodelIDsCheckpoint, modelIDsCheckpoint)
-	// Send a list of delete checkpoints uuids the user wants to delete and
-	// check if it's in model registry.
-	requestedDeleteCheckpoints := []uuid.UUID{checkpoint1.UUID, checkpoint3.UUID}
-	expectedDeleteInModelRegistryCheckpoints := make(map[uuid.UUID]bool)
-	expectedDeleteInModelRegistryCheckpoints[checkpoint1.UUID] = true
-	dCheckpointsInRegistry, err := db.GetRegisteredCheckpoints(requestedDeleteCheckpoints)
-	require.NoError(t, err)
-	require.Equal(t, expectedDeleteInModelRegistryCheckpoints, dCheckpointsInRegistry)
-
 	validDeleteCheckpoint := checkpoint3.UUID
 	numValidDCheckpoints := 1
 
 	require.NoError(t, MarkCheckpointsDeleted(ctx, []uuid.UUID{validDeleteCheckpoint}))
 
 	var numDStateCheckpoints int
-
-	err = db.sql.QueryRowx(`SELECT count(c.uuid) AS numC from checkpoints_view AS c WHERE
-	c.uuid::text = $1 AND c.state = 'DELETED';`, validDeleteCheckpoint).Scan(&numDStateCheckpoints)
+	err = db.Bun().NewRaw(`SELECT count(c.uuid) AS numC from checkpoints_view AS c WHERE
+	c.uuid::text = ? AND c.state = 'DELETED';`, validDeleteCheckpoint).Scan(ctx, &numDStateCheckpoints)
```

Comment:
nit: this is short enough I think you could bunify it

Reply:
done